r/PACKAGING.md

```diff
@@ -0,0 +1,117 @@
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Packaging checklist for CRAN release
+
+For full documentation of the steps in this checklist, see the
+[Apache Arrow Release Management Guide](https://arrow.apache.org/docs/developers/release.html#post-release-tasks).
+
+- [ ] When a release candidate is created
+  [Create a GitHub issue](https://github.com/apache/arrow/issues/new/)
+  entitled `[R] CRAN release version X.X.X` and copy this checklist to the issue
+  
+- [ ] Confirm that all
+  [nightly tests and nightly packaging builds](https://lists.apache.org/list.html?builds@arrow.apache.org) are passing.
+- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html)
```

Comment:
Check them for what?  Some have notes, some have errors. How do I interpret this?

Reply:
What CRAN release branch?

Reply:
Why did we need to pick that commit there? Shouldn't it already be on that branch given it was merged before the RC was cut?

Reply:
Ah wait, patch releases originate from the initial release branch? We should document the difference in expectations here, I didn't know that.  In that case, I should expect that this commit is *not* in the branch as this is a patch release, right?

I just checked the history for the 10.0.1 commit and it is there. Isn't the manual selection of which tickets go into a patch release already sorted (i.e. by Jacob/Raul) by the time we deal with the branch? So only if we wanted to release 10.0.1.1(i.e. some R-specific things) that we'd need to worry about that.

Reply:
> Isn't the manual selection of which tickets go into a patch release already sorted (i.e. by Jacob/Raul) by the time we deal with the branch

Correct, as long as the fix-version (or in the future gh milestone) is set correctly the commits will be cherry picked into the (patch) release branch.